<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Viewport</class>
 <widget class="QWidget" name="Viewport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(128, 128, 128);
color: rgb(255, 255, 255)</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>12</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(179, 255, 184);</string>
       </property>
       <property name="text">
        <string>Camera Label</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="viewport" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(97, 97, 97);</string>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
    </widget>
   </item>
  </layout>
  <action name="aCameraPerspective">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Perspective</string>
   </property>
  </action>
  <action name="aCameraOrtho">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ortho</string>
   </property>
  </action>
  <action name="aCameraLeft">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
  </action>
  <action name="aCameraRight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
  </action>
  <action name="aCameraFront">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Front</string>
   </property>
  </action>
  <action name="aCameraBack">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </action>
  <action name="aCameraTop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Top</string>
   </property>
  </action>
  <action name="aCameraBottom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Bottom</string>
   </property>
  </action>
  <action name="aView1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>View 1</string>
   </property>
  </action>
  <action name="aView2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>View 2</string>
   </property>
  </action>
  <action name="aView3">
   <property name="text">
    <string>View 3</string>
   </property>
  </action>
  <action name="aView4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>View 4</string>
   </property>
  </action>
  <action name="aViewAll">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>All Views</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
